### PR TITLE
thumbnails, date and author improvements

### PR DESCRIPTION
**Pull request by ecnmst**

### CODE REVIEW

File: whotracksme/website/build/blog.py [hunk 1]
```diff
@@ -65,3 +67,14 @@ def build_blogpost_pages(data, blog_posts):
...
     print_progress(text="Generate blog posts")
+
+
+def get_human_date(date):
+    d = datetime.strptime(date, '%Y-%m-%d')
+
+    if 4 <= d.day <= 20 or 24 <= d.day <= 30:
+        suffix = "th"
+    else:
+        suffix = ["st", "nd", "rd"][d.day % 10 - 1]
```
Comment (remusao) [inline whotracksme/website/build/blog.py]: :+1:

File: templates/blog.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
             <div class="row">
                 <div class="col-md-9 col-sm-9">
                     <h1>Blog</h1>
-                    <div class="subtitle">This is our tech blog where we write about privacy, dissect trackers and map the tracking landscape in popular sites.</div>
+                    <div class="subtitle">This a blog space for tech teams at Ghostery and Cliqz.</div>
```
Comment (remusao) [inline templates/blog.html]: "This is" ('is' missing)

Comment (remusao) [reply templates/blog.html]: Same for two occurrences above.

Comment (ecnmst) [reply templates/blog.html]: Thanks - good catch, will update them.

**Pull request merged**